docs/get-started.md

```diff
@@ -157,6 +157,21 @@ Each member file is compressed with a method that must be specified in client co
 In general, not all valid ZIP files are possible to be stream unzipped. However, all files generated by stream-zip are suitable for stream unzipping, for example by [stream-unzip](https://stream-unzip.docs.trade.gov.uk/).
 
 
+## Timestamps
+
+File timestamps (e.g., "modified at") have to fit into the ZIP file format.
+Therefore, stream-zip both rounds and clamps timestamps to make them fit the ZIP file format.
+
+If `extended_timestamps=True` (the default):
+
+ * Timestamps are clamped between 1970-1-1 and 2038-1-19 (both inclusive)
+ * Timestamps are rounded down with 1-second precision
```

Comment:
Is the situation a bit more complex than this? If extended_timestamps=True, then it depends on the client - ones that read the extended timestamps will use the greater range, but older ones will still use the smaller range.

I wonder if it's better if the documentation should be split into "older clients" and "newer clients" (or something like that). The older clients in all cases use the smaller range, but the newer ones depend on extended_timestamps?

Reply:
The data _in the file_ (er, in the stream) doesn't depend on the client, and I think that's all you can reasonably make statements about. What the client will do with it depends on the client, but that doesn't change anything about the stream or the values encoded in it. And documenting what will be done with that data by all possible clients seems both out of scope and impossible.

I guess you could say something like this, to be more precise:

```markdown
If `extended_timestamps=True` (the default), a second
timestamp field is added which will be read by
supporting clients. Extended timestamps:

* Use UNIX-style "seconds since epoch" values
* Are clamped between 1970-1-1 and 2038-1-19
  (both inclusive)
* Are rounded down with 1-second precision
```

Reply:
(Perhaps first documenting the standard timestamps without the "if `extended_timestamps=False`" qualifier, since those timestamps will always be there.)